hr_employee_age/views/hr_employee.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_employee_view_form" model="ir.ui.view">
+        <field name="name">hr.employee.view.form</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form" />
+        <field name="arch" type="xml">
+            <field name="birthday" position="after">
+                <field name="age" />
```

Comment:
In my opinion, `Age 0` should not be displayed if the birthday hasn't been set
```suggestion
                <field name="age" attrs="{'invisible': [('birthday', '=', False)]}" />
```

Reply:
Only be this one, I don't change. Hope you are ok. Just don't want to change the current behavior compare to previous version :).